test/scripts/start-services.sh

```diff
@@ -5,6 +5,7 @@ root_dir="$(realpath ..)"
 parachain_dir="$root_dir/parachain"
 ethereum_dir="$root_dir/ethereum"
 relay_dir="$root_dir/relayer"
+test_dir="$root_dir/test"
```

Comment:
I think its fine for users to call the hardhat task manually to dump the addresses. So lets revert all these changes to the `start-services`.

Reply:
Yes, as suggested, I've adjusted the code.